packages/modular-scripts/package.json

```diff
@@ -8,7 +8,7 @@
   "main": "./dist-cjs/index.js",
   "types": "dist-types/index.d.ts",
   "engines": {
-    "node": ">=12"
+    "node": "^10.13.0 || ^12.13.0 || ^14.15.0 || >=15.0.0"
```

Comment:
`pretty-format` requires a more explicit version of node than we have. This is to ensure we don't get internal errors in subprocesses.

Reply:
Are we sure we want to support 10.x?

Reply:
no.